src/VisualStudio/Core/Def/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -225,6 +226,10 @@ public async Task InitializeUIAffinitizedServicesAsync(IAsyncServiceProvider asy
 
             Logger.Log(FunctionId.Run_Environment,
                 KeyValueLogMessage.Create(m => m["Version"] = FileVersionInfo.GetVersionInfo(typeof(VisualStudioWorkspace).Assembly.Location).FileVersion));
+
+            // Initialize task-list in BG.
+            var taskListService = this.Services.SolutionServices.ExportProvider.GetExports<VisualStudioTaskListService>().Single().Value;
```

Comment:
```suggestion
            var taskListService = this.Services.SolutionServices.ExportProvider.GetExportedValue<VisualStudioTaskListService>();
```

Reply:
i see that only defined in test layer.  do we have a version always available?